python/sglang/srt/server_args.py

```diff
@@ -974,6 +975,11 @@ def add_cli_args(parser: argparse.ArgumentParser):
             action="store_true",
             help="Disable Multi-head Latent Attention (MLA) for DeepSeek V2/V3/R1 series models.",
         )
+        parser.add_argument(
+            "--disable-multimodal",
```

Comment:
Since the issue only occurs when using Llama 4, how about disabling multimodal for Llama 4 by default and then enabling it with --enable-llama4-multimodal. What do you think?

Reply:
We can use it to optimize the performance of all multimodal models. In addition to disabling chunked prefill, multimodal models also reduce `mem_fraction_static` (see [this](https://github.com/sgl-project/sglang/blob/88799448006996eadd137b6bd6559069410e284a/python/sglang/srt/model_executor/model_runner.py#L279)). This argument allows the users to avoid performance degradation.

In addition, I modified the [logger info](https://github.com/sgl-project/sglang/blob/80482d1707b0e8cb92f928c8fda59a2125b311da/python/sglang/srt/model_executor/model_runner.py#L280) to remind the users to turn on this argument.